airflow/models/trigger.py

```diff
@@ -308,3 +308,21 @@ def get_sorted_triggers(cls, capacity: int, alive_triggerer_ids: list[int] | Sel
             skip_locked=True,
         )
         return session.execute(query).all()
+
+    @classmethod
+    @internal_api_call
+    @provide_session
+    def filter_out_reassigned_triggers(
```

Comment:
function should be renamed. it's not filtering them out -- it's "getting them"

Reply:
i.e. it returns the list of reassigned triggers. something _else_ might filter out but this does not.

Reply:
Thanks, I already renamed to get_reassigned_triggers